### PR TITLE
TET-15 Upgrade to webpack 5

### PR DESCRIPTION
**Pull request by craigjbass**
## Description of change

_Document what the PR does and why the change is needed_

## Test instructions

_What should I see?_

### Before

_Add a screenshot_

### After

_Add a screenshot_

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,63 @@
-FROM gcr.io/sre-docker-registry/data-hub-frontend-dependencies:1.0.7
+FROM node:14.17.0-buster
```
Comment (craigjbass) [inline Dockerfile:1]: 1.0.7 has the changes you seek. You can revert this to the version in default branch

Comment (vfarah-if) [reply Dockerfile]: Fixed

File: test/functional/cypress/specs/investments/opportunity-interactions-spec.js [hunk 2]
```diff
@@ -103,7 +103,8 @@ describe('The interactions tab on an opportunity page', () => {
       cy.get('[data-test="sortby"] select').select('date')
       assertQueryParams('sortby', 'date')
     })
-    it('should allow sort by recently created interactions', () => {
+    it('should allow sort by recently created interactions when oldest selected first', () => {
+      cy.get('[data-test="sortby"] select').select('date')
       cy.get('[data-test="sortby"] select').select('-date')
       assertQueryParams('sortby', '-date')
     })
```
Comment (vfarah-if) [inline test/functional/cypress/specs/investments/opportunity-interactions-spec.js:110]: Delete the test as this is the default sort and is therefore not set in the url now

File: test/unit-client/assets/javascripts/modules/add-items.test.js [hunk 3]
```diff
@@ -433,9 +433,7 @@ describe('Add another', function () {
           data-item-selector=".c-form-fieldset"
           data-add-button-selector="#add"
           >
-
           ${fieldset}
-
           <p>
```
Comment (vfarah-if) [inline test/unit-client/assets/javascripts/modules/add-items.test.js:437]: Rollback this merge issue

**Review by vfarah-if — APPROVED** [review]

**Review by santoshdasa12345 — APPROVED** [review]

**Pull request merged**